vtex/admin-promotions
# Set table back to first page after search

#### What is the purpose of this pull request?
Fix table showing no results when out of table's first page.

#### How should this be manually tested?
BEFORE: https://pricingqa.myvtex.com/admin/promotions
NOW (beta): https://vlauxbeta--pricingqa.myvtex.com/admin/promotions
- Open promotions list
- Navigate to page 3
- Search for "teaser"

Expected: to find at least one result

#### Types of changes
